app/templates/eventTemplates/all-auditoriums.html

```diff
@@ -0,0 +1,22 @@
+{% extends "eventTemplates/layout.html" %}
+{% block title %}Wszystkie sale{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style>
+    </style>
+{% endblock %}
+{% block body %}
+    <h1>Wszystkie sale</h1>
+    <table class=todos>
+      <tr>
+        <th>#
```

Comment:
Nie wiem czy wam to IDE poprawia ale domykajcie te tagi.
```suggestion
        <th>#</th>
```